core/services/chainlink/config_general.go

```diff
@@ -345,6 +345,17 @@ func (g *generalConfig) StarkNetEnabled() bool {
 	return false
 }
 
+// TODO: Update this method once Aptos becomes public integration
```

Comment:
What is left to update?

Reply:
thanks for pointing this out @jmank88 . Removing this from the code